src/app/public/modules/inline-form/inline-form.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="sky-inline-form sky-shadow">
+  <div class="sky-inline-form-content">
+    <ng-content></ng-content>
+  </div>
+  <div class="sky-inline-form-footer">
+    <button
+      class="sky-btn sky-btn-primary"
+      (click)="onDoneClick()"
+    >
+      {{ 'skyux_inline_form_done' | skyLibResources }}
+    </button>
+    <button *ngIf="showDeleteButton"
+      class="sky-btn sky-btn-secondary"
```

Comment:
This button is showing up transaparent, should be white.

Reply:
Good catch, I was using the wrong class. Fixed.